### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ptbrzoska**
1. updated diagram
2. updated getting started commands

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ The setup contains following demo datasets:
 
 ### Running the example
 In order to run it, you need to:
-- clone this repository: `git clone git@github.com:QuesmaOrg/quesma.git`
+- clone this repository: `git clone https://github.com/QuesmaOrg/quesma.git`
 - run `docker-compose up` directly from `/examples/kibana-sample-data` directory
```
Comment (mieciu) [inline README.md:57]:
```suggestion
- run `docker-compose up` directly from `examples/kibana-sample-data` directory
```

**Review by mieciu — APPROVED** [review]
one nit spotted (unrelated to your changes, but worth adding if we're already working on this), other than that LGTM

File: README.md (hunk 1 above)
Comment (avelanarius) [inline README.md:57]:
```suggestion
- `cd quesma/examples/kibana-sample-data`
- run `docker-compose up`
```

or maybe like this? This way it's easily copy-paste'able

Comment (ptbrzoska) [reply README.md]: Let's leave it as it was and have ppl to decide how they want to do it. If someone needs a command - it is provided below.

**Pull request merged**